### DISCUSSION TITLE
Windows Build Error

### DISCUSSION THREAD

**Discussion by gnusec** [moonD4rk/HackBrowserData#268] [Q&A] [answered]
**Describe the bug**

> set CGO_ENABLED=1
> set GOOS=windows
> set GOARCH=amd64
> set CC=x86_64-w64-mingw32-gcc
> go build
error info:
no Go files in D:\HackBrowserData

**Desktop (please complete the following information):**
 - OS Name: Windows 10
 - Browser Name:
 - Browser Version:

**Comment by sh3d0ww01f** [comment, marked as answer]
go build ./...
